src/common/pf_lldp.h

```diff
@@ -43,6 +43,19 @@ void pf_lldp_init(
 void pf_lldp_send(
    pnet_t                  *net);
 
+/**
+ * Receive an LLDP message.
+ *
+ * Parse LLDP tlv format and store selected information.
+ * @param net              InOut: The p-net stack instance
+ * @param p_buf            In:    The Ethernet frame.
+ * @param frame_id_pos     In:    The position of the frame id.
```

Comment:
The parameter is named frame_pos, offset and frame_id_pos here and in the .c file. Make it consistent. Make sure it is the position of the frame id that actually is inserted.
Also p_buf is inconsistently named.